tmt/beakerlib.py

```diff
@@ -125,19 +146,26 @@ def __init__(self, identifier, parent=None):
             raise tmt.utils.SpecificationError(
                 f"Repository '{self.url}' does not contain fmf metadata.")
 
-    def __str__(self):
+    def __str__(self) -> str:
         """ Use repo/name for string representation """
         return f"{self.repo}{self.name}"
 
-    def fetch(self):
+    @property
+    def _library_cache(self) -> Dict[str, Library]:
+        if not hasattr(self.parent, '_library_cache'):
+            self.parent._library_cache = dict()
+
+        return self.parent._library_cache
+
+    def fetch(self) -> None:
         """ Fetch the library (unless already fetched) """
         # Initialize library cache (indexed by the repository name)
         if not hasattr(self.parent, '_library_cache'):
             self.parent._library_cache = dict()
```

Comment:
These two lines are now part of `_library_cache` property, therefore can be dropped.

Reply:
ok